Various tweaks

birkirb opened the pull request:
Hi @lucasnevespereira !

First off, many thanks for putting your code out there, very nicely done! :clap: 

I made some adjustments that you may or may not like, but thought I'd present them here if there is something you do like.

The main changes are the following:
* Changing the locales to simple (i.e. en instead of en_US). This is mainly since the translations are very simple, and most `jsonresumes` will default to that. The locale standards suggest that en_US/en_CA etc should fallback to "en" when region specific locales are not in place, but just going for the simple option seemed best.
* Modifying the modern template. I liked it a lot, but added some tweaks to group duplicate companies and the LinkedIn way of presenting that. Also adjusted default pixel sizes for html and scaled down for the PDF generation as I felt the html font size didn't fit the page as nicely. This will mess up other templates, I'm sure. 
* Moving the locales and templates into their own sub "ui" directories and adding an optional argument for it. This will allow folks to have their resume building outside of the project (i.e. use the binary and local command). Mostly the translation file placement made this impossible.
* Default to the local command if no command argument is passed but others are (i.e. you can use just `-f`)

Happy to split these up if you like and you can of course edit out what you do not like. 
Also feel free to close without further comment, I know life is busy and this may just extra work for you. :peace_symbol: :v:

birkirb commented:
Agree with everything you said, and totally understand about the scale thing as you would have to update all the other templates.

Let me look at the logo header stuff. Not completely sure I understand what you mean, but guess it's about putting the title up front when it's not a grouped entry, and I can see how in general it would focus on the company rather than the position, which is indeed not the norm.

lucasnevespereira commented:
About the logo header, yes I just got what you intent to do! And yes indeed I just wanted to handle the case when it is not multiple grouped positions inside the same company, to have a summary and highlights displaying correctly. But if it is too much work just leave it as you see fit don't worry much about it!

birkirb commented:
Alright, I've removed the scale, pulled in your changes and I adjusted the fonts to emphasize the position rather than the company. 

I blew my time-box on making the single positions flip the company/title order vs the group ones.
Given that I might be good for now, and I guess you can decide if you want to migrate the modified "modern" into it's own "linkedin" or not. If you do I'll pull it in, and maybe later look into how I can have the different cases handled.

Hope this is good for now.

lucasnevespereira commented:
Hi @birkirb, I've had somme UI issues when generating the template with the example resume `make local file="examples/example.resume.json"`, I don't know if this is the style you were looking for in skills.

![Screenshot 2025-03-27 at 11 36 13](https://github.com/user-attachments/assets/bcfd5a98-251b-4ddb-b639-8d09cc87d527)

What I am going to do is merge this and then separate modern from linkedin template so we keep the old modern template and you can keep applying changes to your linkedin template if needed!

Thanks again for your contribution 🙂
